app/helpers/metadata_helper.rb

```diff
@@ -4,7 +4,7 @@ module MetadataHelper
   include ErrorHelper
 
   def get_available_matching_field(sample, name)
-    available_fields = (sample.project.metadata_fields & sample.host_genome.metadata_fields)
+    available_fields = sample.project.metadata_fields
```

Comment:
so are the host genome fields already included in the project fields?

Reply:
We are making the available fields less restrictive. Before, it was fields that match both the project and host_genome. Now, it's just fields that match the project.

This means that fields on the project but not compatible with the host genome will now be returned, which is part of the intention of this PR. We want the upload process to attempt to use metadata fields that are incompatible with the host genome and fail, instead of ignoring these metadata fields and creating a confusing custom field.